### PR TITLE
Update GPU install Ubuntu 16.04

### PR DESCRIPTION
**Pull request by lamberta**
Clarify that CUDA Ubuntu 16.04 Install is for version 10.1
Add `libnvinfer-plugin6=6.0.1-1+cuda10.1` for TensorRT

This PR supersedes  https://github.com/tensorflow/docs/pull/1403 since I can't seem to add commits there.

### CODE REVIEW

**Comment by lamberta** [comment]
Also tracking internally: b/148153266

**Comment by sarthfrey** [comment]
Do we want `libnvinfer5` or `libnvinfer6`?

**Comment by gunan** [comment]
Great question. @chsigg @sanjoy which libnvinfer does TF use right now?

File: site/en/install/gpu.md [hunk 1]
```diff
@@ -141,14 +141,16 @@ complicates installation of the NVIDIA driver and is beyond the scope of these i
...
+    cuda-10-1 \
     libcudnn7=7.6.4.38-1+cuda10.1  \
     libcudnn7-dev=7.6.4.38-1+cuda10.1
 </code>
 
 # Install TensorRT. Requires that libcudnn7 is installed above.
-<code class="devsite-terminal">sudo apt-get install -y --no-install-recommends libnvinfer5=6.0.1-1+cuda10.1 \
-    libnvinfer-dev=6.0.1-1+cuda10.1
+<code class="devsite-terminal">sudo apt-get install -y --no-install-recommends \
+    libnvinfer5=6.0.1-1+cuda10.1 \
```
Comment (aaroey) [inline site/en/install/gpu.md]: Should be libnvinfer6=xxx.

**Comment by lamberta** [comment]
Thank you @sarthfrey for the update and thanks @gunan for the review!

**Pull request merged**